app/uk/gov/hmrc/tai/service/TaxCodeChangeService.scala

```diff
@@ -35,8 +35,8 @@ class TaxCodeChangeService @Inject()(taxCodeChangeConnector: TaxCodeChangeConnec
 
   def taxCodeChange(nino: Nino)(implicit hc: HeaderCarrier): Future[TaxCodeChange] =
     taxCodeChangeConnector.taxCodeChange(nino) map {
-      case TaiSuccessResponseWithPayload(taxCodeChange: TaxCodeChange) => taxCodeChange
-      case _                                                           => throw new RuntimeException(s"Could not fetch tax code change")
+      case taxCodeChange: TaxCodeChange => taxCodeChange
+      case _                            => throw new RuntimeException(s"Could not fetch tax code change")
```

Comment:
as we have moved this to the connector we can remove the case _ and therefore the whole match statement